modules/core/src/main/java/org/apache/synapse/deployers/LocalEntryDeployer.java

```diff
@@ -79,6 +102,66 @@ public String deploySynapseArtifact(OMElement artifactConfig, String fileName,
         return null;
     }
 
+    private void handleHttpConnectorCertificates(OMElement element) throws DeploymentException {
```

Comment:
This doesn;t have to be specific to http connector right? If so shall we change the method name?

Reply:
Updated with `handleSSLSenderCertificates`.